src/Distill.php

```diff
@@ -104,7 +104,7 @@ public function extractText(ItemCollection $items)
             ->map(fn ($value) => match ($value->fieldtype()->handle()) {
                 'text' => $value->value(),
                 'textarea' => $value->value(),
-                'bard' => Statamic::modify($value)->bardText()->fetch(),
```

Comment:
Actually I'll probably revert this now the modifier can deal with strings (and will do `strip_tags` too), unless you think it's needed here as well?